frontend/src/pages/protocol-token/list/all.vue

```diff
@@ -0,0 +1,274 @@
+						Complete list of all available PLT tokens on the network
+					</p>
+				</div>
+				<NuxtLink
+					to="/protocol-token"
+					class="text-sm text-theme-text-secondary hover:text-theme-interactive transition-colors duration-200 flex items-center gap-1"
+				>
+					<svg
+						class="w-4 h-4"
+						fill="none"
+						stroke="currentColor"
+						viewBox="0 0 24 24"
+					>
+						<path
+							stroke-linecap="round"
+							stroke-linejoin="round"
+							stroke-width="2"
+							d="M15 19l-7-7 7-7"
+						></path>
+					</svg>
+					<span>Back to Overview</span>
+				</NuxtLink>
+			</header>
+
+			<!-- Tokens List -->
+			<div
+				class="bg-theme-background-primary rounded-lg border border-theme-border overflow-hidden"
+			>
+				<div class="overflow-x-auto">
+					<table class="w-full">
+						<thead
+							class="bg-theme-background-secondary border-b border-theme-border"
+						>
+							<tr>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Sl. No.
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Token
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Token ID
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Total Supply
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Minted
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Burned
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Holders
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Issuer
+								</th>
+								<th
+									class="text-left py-3 px-4 font-medium text-theme-text-secondary"
+								>
+									Created
+								</th>
+							</tr>
+						</thead>
+						<tbody>
+							<!-- Loading State -->
+							<TableRow v-if="pltTokenLoading">
+								<td colspan="9" class="text-center py-8">
+									<LoadingIndicator />
+								</td>
+							</TableRow>
+
+							<!-- Empty State -->
+							<TableRow v-else-if="!pltTokenData?.length">
+								<td
+									colspan="9"
+									class="text-center py-8 text-theme-text-secondary"
+								>
+									No tokens available.
+								</td>
+							</TableRow>
+
+							<!-- Token Rows -->
+							<TableRow
+								v-for="(token, index) in pltTokenData"
+								:key="token.tokenId"
+								class="hover:bg-theme-background-secondary transition-colors duration-200"
+							>
+								<td class="py-4 px-4">
+									<span class="text-sm font-medium text-theme-text-secondary">
+										{{ (currentPage - 1) * pageSize + index + 1 }}
```

Comment:
To avoid calculating/number the tokens as this value seems the same as the `token.index`.
```suggestion
										{{ token.index }}
```